diamm/serializers/website/stats.py

```diff
@@ -0,0 +1,11 @@
+import serpy
+
+from diamm.serializers.serializers import ContextDictSerializer
+
+class StatsSerializer(ContextDictSerializer):
+    source = serpy.IntField(label='sources')
```

Comment:
This pattern is used all over the place in the other serializers: https://github.com/diamm/diamm/blob/master/diamm/serializers/search/source.py#L24